data/data.go

```diff
@@ -5,17 +5,28 @@ import (
 	"fmt"
 )
 
+// ListKey represents the primary key of a list
+type ListKey struct {
+	ID string `json:"Id"`
+}
+
 // List represents the data structure of a list
 type List struct {
-	ID   string `json:"Id"`
+	ListKey
 	Name string `json:"Name"`
 }
 
-// Item represents the data structure of an item on a list
-type Item struct {
+// ItemKey represents the primary key of an item
+type ItemKey struct {
 	ID     string `json:"Id"`
 	ListID string `json:"ListId"`
-	Name   string `json:"Name"`
+}
+
+// Item represents the data structure of an item on a list
+type Item struct {
+	ItemKey
```

Comment:
I've moved the primary key to its own struct so it can be used independently, and just re-used it here